docs/src/components/AutoCompleteExample/CustomResults.component.tsx

```diff
@@ -1,37 +1,50 @@
 import * as React from 'react';
 import { ListItem, Check, Typography } from '@retailmenot/anchor';
 
-interface Values {
-    [key: string]: any;
+type Value = {
+    active: boolean;
+    id: string;
+    key: string;
+    label: string;
+    onMouseOver: (event?: React.MouseEvent) => void;
+    onSelect: (event?: React.FocusEvent) => void;
+    [key: string]: any; // Value also contains any new fields passed via the dataSource prop
```

Comment:
Replacing `new fields` with something like `additional custom fields` makes more sense to me.